packages/fast-components-react-base/src/listbox/listbox-context.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+export interface ListboxItemData {
+    id: string;
+    value: string;
+    displayString: string;
+}
```

Comment:
Curious on the reason to create a new interface here for this data rather than expecting the prop contract itself. At a glance it seems like passing ListboxItemProps would give us a more extensible solution long-term, but I may be missing an obvious reason for a different interface

Reply:
Synced with @nicholasrice on his thoughts here and I don't think we'll run into perf issues by passing the props.

Reply:
Pushed an update that uses props.  Seems to work as expected.